web/app/Style.js

```diff
@@ -81,6 +81,9 @@ Ext.define('Traccar.Style', {
     mapGeofenceWidth: 5,
     mapGeofenceRadius: 9,
 
+    mapAnimateMarkerDuration: 2000,
+    mapAnimateMarkerTimoeout: 15,
```

Comment:
Typo

Reply:
Also, updating every 15 ms seem too often. It can cause issues when you have many markers. Maybe something between 30-50 is more reasonable.